packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -6198,3 +6232,27 @@ The application CSR object defines a certificate signing request for a signature
 | csr              | Base64 encoded CSR in DER format                             | String                                                                      | TRUE     | TRUE   | TRUE     |           |           |            |
 | id               | unique identifier for the CSR                                | String                                                                      | FALSE    | TRUE   | TRUE     |           |           |            |
 | kty              | cryptographic algorithm family for the CSR's keypair         | String                                                                      | FALSE    | FALSE  | TRUE     |           |           |            |
+
+### ACS Endpoint Object
+
+The ACS endpoint containing the ACS URI and index of the URI.
+
+```json
+{
+  "url": "https://www.example.com/sso/saml",
+  "index": 0
+}
+```
+
+
+#### ACS Endpoint properties
+
+| Property         | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly | MinLength | MaxLength | Validation |
+| ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- | --------- | --------- | ---------- |
+| url              | URL of the ACS                                               | String                                                                      | FALSE    | FALSE  | FALSE    |           | 1024      | [URL](http://tools.ietf.org/html/rfc3986)           |
+| index            | index of the URL in the array of ACS endpoints               | Number                                                                      | FALSE    | TRUE   | FALSE    | 0         |           |            |
+
+Property details
+
+ * `url` cannot have query or fragment parameters.
```

Comment:
we try to keep a "conversational tone" - so we use contractions:

* 'url' can't have query or fragment parameters.

Reply:
done